docs/resources/service_account_key.md

```diff
@@ -42,6 +42,7 @@ resource "twingate_service_account_key" "github_key" {
 ### Read-Only
 
 - `id` (String) Autogenerated Service Key ID
```

Comment:
For the `id` value, this is the Terraform ID, correct? So this should be "The ID of the Service Key resource" since it refers to the TF ID.